includes/template-functions.php

```diff
@@ -876,6 +892,17 @@ function sensei_the_single_lesson_meta() {
 		?>
 		<section class="lesson-meta">
 			<?php
+			/**
+			 * Filters the position of the video in the lesson meta.
+			 *
+			 * Fires before the lesson meta is displayed.
+			 *
+			 * @hook sensei_video_position
+			 *
+			 * @param {string} $position  The position of the video in the lesson meta, default: top.
+			 * @param {int}    $lesson_id The lesson ID.
+			 * @return {string} The position of the video in the lesson meta.
+			 */
 			if ( apply_filters( 'sensei_video_position', 'top', get_the_ID() ) == 'bottom' ) {
 
 				do_action( 'sensei_lesson_video', get_the_ID() );
```

Comment:
Should we also add a comment here saying that it's documented on `templates/single-lesson.php`?

Reply:
I didn't work on actions in this PR :)
With the next PR I'm going to cover actions 👍